geopandas/io/arrow.py

```diff
@@ -280,12 +285,25 @@ def _to_parquet(df, path, index=None, compression="snappy", version=None, **kwar
         "pyarrow.parquet", extra="pyarrow is required for Parquet support."
     )
 
+    if kwargs and "version" in kwargs and kwargs["version"] is not None:
+        warnings.warn(
+            "the `version` parameter has been replaced with `schema_version` "
+            "and will be removed in a future release.  It will instead be "
```

Comment:
We won't _remove_ it, we will only no longer use it for `schema_version`, but we should still pass it through to pyarrow parquet writer so you can specify the Parquet version.

Reply:
Yes, that sounds good